system-addon/content-src/styles/_theme.scss

```diff
@@ -7,6 +7,7 @@ body {
   --newtab-button-primary-color: $blue-60;
   --newtab-button-secondary-color: inherit;
   --newtab-element-active-color: $grey-20;
+  --newtab-element-hover-color: $grey-20;
```

Comment:
There should be no differentiation from hover and clicking on a context menu item for default theme?

Reply:
Oh right, it needs to get darker. 👍